src/main/scala/tilelink/Bus.scala

```diff
@@ -26,19 +27,24 @@ abstract class TLBusWrapper(params: TLBusParams)(implicit p: Parameters) extends
   val masterBuffering = params.masterBuffering
   val slaveBuffering = params.slaveBuffering
   require(blockBytes % beatBytes == 0)
+  private val delayProb = p(TLBusDelayProbability)
 
+  private val delayer = if (delayProb > 0.0) Some(LazyModule(new TLDelayer(delayProb))) else None
```

Comment:
@terpstra told me a better source of entropy: TLDelayer(delayProb)(TLBuffer(BufferParams.flow)(TLDelayer(delayProb)))